charts/aspnetcore/values.yaml

```diff
@@ -66,9 +66,19 @@ ingress:
 
 ## ASP.NET Core deployment parameters
 ## @param deployment.name Name of the deployment, automatically generated from the release name if not specified
+## ref: https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/
+## @param deployment.autoscaling.enabled Default to false. Enable autoscaling
+## @param deployment.autoscaling.minReplicas Optional if PodDistributionBudget is set. Minimum number of ASP.NET Core replicas minimum is PodDisruptionBudget.minAvailable + 1
+## @param deployment.autoscaling.maxReplicas Optional if PodDistributionBudget is set or minReplicas is set. Maximum number of ASP.NET Core replicas minimum is minReplicas value.
+## @param deployment.autoscaling.targetCPUUtilizationPercentage Target CPU utilization percentage for autoscaling
 ##
 deployment:
   name: ""
+  autoscaling:
```

Comment:
Any reason why we'd add this one under deployment? 

In terms of uniformity I'd leave it at root like the others. It's also how others seem to do it https://github.com/bitnami/charts/blob/main/bitnami/aspnet-core/values.yaml

Reply:
it was just to be explicit about the relationship of one deployment <-> one hpa